nomad/constants.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 
 
+# PROJECTS
+# --
+
+METADATA_BASE_DIR = '.nomad'
```

Comment:
Hmm, perhaps we could use another strategy to ensure that our LXD name is unique (and well known by lxd-nomad).

Instead of creating a name using the patterm "{project_slug}-{container_name}-{id}" with a generated ID (which forces us to store it inside a "metadata" hidden file), we could use the same pattern with an ID corresponding to the inode number of the folder corresponding to the "homedir" of the project, thus we could be sure that the LXD name is unique.